README.md

```diff
@@ -151,18 +151,33 @@ In [firebase console](https://console.firebase.google.com/), you can get `google
 
 ## Usage
 
+### Receiving Notifications
 ```javascript
 import FCM from 'react-native-fcm';
 
 class App extends Component {
   componentDidMount() {
     FCM.requestPermissions(); // for iOS
+    
+    FCM.getInitFCMData().then((initialData)=>{
+      /* initialData will not be null if :
+      1- Android, app in background when notification received
+      2- Android, app killed when notification received
+      3- iOS, app killed when notification received
+      */
+    })
+    
     FCM.getFCMToken().then(token => {
       console.log(token)
       // store fcm token in your server
     });
     this.notificationUnsubscribe = FCM.on('notification', (notif) => {
-      // there are two parts of notif. notif.notification contains the notification payload, notif.data contains data payload
+      /*
+      This function will be called if :
+        1- Android and app is in the foreground ( notif['opened_from_tray'] will be set to false )
```

Comment:
FCM.on('notification') fires in iOS if notification is received when app is active or in background and the user clicked on the notification.
I wanted to be able to distinguish between these two. 
If the notification event is triggered from a user interaction then I would need to do some custom action(routing for example), if it was triggered without user interaction than I'd need to be aware of that to do another type of action (set notification in app, push local notification ...)

This would help with #88 too.